python/adjoint/utils.py

```diff
@@ -149,18 +115,20 @@ def install_design_region_monitors(
     simulation: mp.Simulation,
     design_regions: List[DesignRegion],
     frequencies: List[float],
-    decimation_factor: int = 0
+    decimation_factor: int = 0,
 ) -> List[mp.DftFields]:
     """Installs DFT field monitors at the design regions of the simulation."""
-    design_region_monitors = [
+    design_region_monitors = [[
         simulation.add_dft_fields(
-            _compute_components(simulation),
+            [comp],
             frequencies,
             where=design_region.volume,
             yee_grid=True,
-            decimation_factor=decimation_factor
-        ) for design_region in design_regions
-    ]
+            decimation_factor=decimation_factor,
+            persist=True,
+            expand=True
```

Comment:
Why do these values override the defaults where `persist` and `expand` are both `False`? Maybe just change the defaults to `True`?

Reply:
Because we don't want the default to be true.